charts/miggo-operator/templates/clusterrole.yaml

```diff
@@ -73,6 +73,18 @@ rules:
       - patch
       - update
       - watch
+  - apiGroups:
+      - ""
```

Comment:
This wont make us problems with dev-ops guys ? Gets us access to all the cluster secrets no?

Reply:
adding feature flag